docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -72,3 +72,17 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+ifeval::["{build}" != "downstream"]
+[WARNING]
+In IPv6, {OpenShift} worker nodes need a `/64` prefix allocation due to OVN
```

Comment:
```suggestion
In IPv6, {rhocp_long} worker nodes need a `/64` prefix allocation due to OVN
```

@elvgarrui I recently changed the "OpenShift" variable to match downstream. See above.

Reply:
Done!